.github/workflows/editorconfig.yml

```
@@ -17,4 +17,20 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npx editorconfig-checker
+      - id: files
+        uses: jitterbit/get-changed-files@v1
+      - name: Проверка линтером
+        run: |
+          npm install editorconfig-checker -g
```

```suggestion
          npm ci editorconfig-checker
```
Я могу ошибаться, но кажется в CI нужно (можно?) так

---

@pepelsbey не в этом случае. Тут зависимость ставится внутрь экшна. Её нет в проекте, также как и до этого было сделано. `npx` как раз выкачивает и применяет. Поэтому включать в сборку не надо.

---

В этом случае ci быстро поставит зависимость (быстрее чем i) и -g не нужен, ведь папка будет тут же, где скрипт запускается, пакет будет видно. Но может я чего-то не понимаю :)

---

Проверил, действительно работает. Давай попробуем.

---

@pepelsbey а вот и не заработало. В документации есть требование: "The project must have an existing package-lock.json or npm-shrinkwrap.json." А у нас же нет его. Странно, что собралось у меня в `dist`... Возвратил.

---

Ну тогда мы использовали `npx editorconfig *.md` до сих пор, что тоже проще и быстрее, кажется.

---

> Ну тогда мы использовали `npx editorconfig *.md` до сих пор, что тоже проще и быстрее, кажется.

Да, но в этом случае для каждого файла сначала скачивается `editorconfig-checker`, а потом уже идёт проверка.

---

Тогда всё правильно :) Извини, что отвлёк, хотелось разобраться )

---

Конечно! Наоборот, проверили всё на тему глубины познаний :)